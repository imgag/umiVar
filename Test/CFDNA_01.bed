chr1	115256330	115256731
chr1	160786470	160786871
chr1	233448213	233448614
chr1	47395674	47396075
chr11	5708828	5709229
