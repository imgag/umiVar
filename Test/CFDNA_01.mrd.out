#MRD_log10	MRD_pval	SUM_DP	SUM_ALT	Mean_AF	Median_AF
20.0	1.00e-20	24544	1599	0.065148	0.046232
